application/src/main/java/org/togetherjava/tjbot/commands/tag/TagUtility.java

```diff
@@ -0,0 +1,68 @@
+package org.togetherjava.tjbot.commands.tag;
+
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.entities.Emoji;
+import net.dv8tion.jda.api.entities.MessageChannel;
+import net.dv8tion.jda.api.entities.MessageEmbed;
+import net.dv8tion.jda.api.events.interaction.SlashCommandEvent;
+import net.dv8tion.jda.api.interactions.components.Button;
+import net.dv8tion.jda.api.interactions.components.ButtonStyle;
+import org.togetherjava.tjbot.tag.TagSystem;
+
+import java.awt.*;
+import java.time.LocalDateTime;
+
+public class TagUtility {
```

Comment:
missing javadoc. what does the class offer? give some overview please

Reply:
done